src/site/content/en/blog/profiling-web-audio-apps-in-chrome/index.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Profiling Web Audio in Chrome
+subhead: Learn how to profile the performance of Web Audio apps in Chrome using chrome://tracing and the WebAudio panel in DevTools.
+description: Learn how to profile the performance of Web Audio apps in Chrome using chrome://tracing and the WebAudio panel in DevTools.
+date: 2020-04-30
+updated: 2020-04-30
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - media
+  - webaudio
+authors:
+  - hongchanchoi
```

Comment:
We should add a hero image or else the post won't show up in the **Latest Stories** section of the homepage.

https://web.dev/handbook/markup-media/#hero

We get most of our hero images from unsplash.com

Reply:
added